content/02.body.md

```diff
@@ -1,13 +1,23 @@
 ## Introduction
 
-Networks contain information about relationships between entities ("edges between nodes").
-A node's degree is the number of relationships it has in the network.
+Networks contain information about relationships between entities (referred to here as "edges" between "nodes").
+A node's degree is the number of edges it has in the network.
+`TODO: do we need to discuss in-degree vs out-degree and hetnet degree here?`
 Networks contain many nodes, whose degrees can be aggregated to form the network's degree distribution.
-Because different nodes can have very different degrees, real networks have highly variable degree distributions (Figure {@fig:hetionet}).
+Because different nodes can have very different degrees, real networks generally have highly variable degree distributions [@doi:10.1126/science.286.5439.509; @doi:10.1038/s41467-019-08746-5; @doi:10.1371/journal.pone.0017645].
+This is especially true for networks encoding biomedical knowledge or assays — where natural forces such as preferential attachment inherent to the problem domain combine with observation-based influences such as study methodology — to create non-uniform degree distributions (Figure {@fig:hetionet}).
```

Comment:
Should there be spaces between the dashes? 

```suggestion
This is especially true for networks encoding biomedical knowledge or assays—where natural forces such as preferential attachment inherent to the problem domain combine with observation-based influences such as study methodology—to create non-uniform degree distributions (Figure {@fig:hetionet}).
```

Reply:
On second thought, what do you think about the following punctuation?

```suggestion
This is especially true for networks encoding biomedical knowledge or assays, where natural forces such as preferential attachment inherent to the problem domain combine with observation-based influences such as study methodology to create non-uniform degree distributions (Figure {@fig:hetionet}).
```

Reply:
All suggested changes are fine are purely stylistic choices.